lib/functional_mixin.js

```diff
@@ -187,6 +187,41 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.multipleCheckbox = function busterFunctionalMixin_multipleCheckbox(target, callback)
+  {
+    target = typeof target == 'string' ? this.$(target) : target;
+    // update target
+    function selectCheckbox(target)
```

Comment:
it's a little strange, imo, that you pass in an array and it only uses the first element. Maybe just pass in the first element?